nix/build/ocaml-overlay.nix

```diff
@@ -17,7 +16,7 @@ with opam-nix.lib.${pkgs.system}; let
     resolveArgs = { };
   } tezosSourcesResolved { };
 in {
-  ocamlPackages = tezosScope.overrideScope' (pkgs.lib.composeManyExtensions [
+  ocamlPackages = (tezosScope.overrideScope' (pkgs.lib.composeManyExtensions [
```

Comment:
Why not just `tezosPackages = ...` here and then not have the awkward `// super.ocamlPackages` ?